src/common/src/session_config/mod.rs

```diff
@@ -128,14 +141,17 @@ pub struct ConfigMap {
 
     /// If `STREAMING_PARALLELISM` is non-zero, CREATE MATERIALIZED VIEW/TABLE/INDEX will use it as
     /// streaming parallelism.
+    #[serde_as(as = "DisplayFromStr")]
     #[parameter(default = ConfigNonZeroU64::default())]
     streaming_parallelism: ConfigNonZeroU64,
 
     /// Enable delta join for streaming queries. Defaults to false.
+    #[serde(rename = "rw_streaming_enable_delta_join")]
```

Comment:
Do we have to rename here? There's no compatibility concern for serializing the session config map.

Reply:
I originally thought we need read from config file. We can remove this.

Reply:
Second thought, it won't hurt to keep this.

Reply:
Still don't quite get that. Why do we need the rename? Note that `rename` on `parameter` attribute is more like an alias and is user-facing, but `serde` is not.

Reply:
Considering the meta stored the renamed name as the key, it won't hurt to keep everything consistent, including the encoded json.

Reply:
Removed